readme.md

```diff
@@ -59,30 +63,32 @@ How can we tell this actually created the database? Run the following in the ter
 $ rails db
 ```
 
-Rails also gives us a very helpful way to load up all the models in our application, and play with them in the console:
+Rails also gives us a very helpful way to load up all the models in our application, and play with them in the console...
 
 ```bash
 $ rails c
 ```
 
 #### What does the `rails` command do?
 
-When developing a rails application, developers frequently encounter very similar types of problems, i.e. setting up a database, changing the name of a column, or seeding a database. Luckily, Rails as a framework has several built in commands / helpers that allow us to implement several repetitive command line tasks into a  single command.  
+When developing a rails application, developers frequently encounter very similar types of problems, i.e. setting up a database, changing the name of a column or seeding a database. Luckily, Rails has several built in commands / helpers that allow us to implement repetitive command line tasks into a single command.  
 
 The `rails` command is frequently used to...
-* Create the database (`rails db:create`)
-* Create / Edit database tables (`rails db:migrate`)
-* Drop the database (`rails db:drop`)
-* Seed the database (`rails db:seed`)
+* Create the database - `rails db:create`
+* Create and edit database tables - `rails db:migrate`
+* Drop the database - `rails db:drop`
+* Seed the database - `rails db:seed`
 
-[Learn more about rails command here](https://github.com/ruby/rails#description).  
+> [Learn more about the `rails` command here.](https://github.com/ruby/rails#description).  
 
 ## You Do: Models (10 minutes / 0:20)
 
-**[5 minutes]** Create Artist and Song models the same way you did in last week's Active Record class. They should...
-* Have the appropriate file name.
-* Have the appropriate model name.
-* Inherit from Active Record.
+> 5 minutes exercise. 5 minutes review.
+
+Create `Artist` and `Song` models the [same way you did in last week's Active Record class](https://github.com/ga-wdi-lessons/activerecord-intro#defining-our-models). They should...
```

Comment:
Actually that section in the Sinatra AR lesson says to leverage old code that isn't directly referenced in that lesson plan. Going to stick with Active Record LP since the code is there.